docs/src/modules/components/AppContainer.js

```diff
@@ -17,5 +17,5 @@ const StyledContainer = styled(Container)(({ theme }) => {
 });
 
 export default function AppContainer(props) {
-  return <StyledContainer id="main-content" maxWidth={false} tabIndex={-1} {...props} />;
+  return <StyledContainer id="main-content" maxWidth={false} {...props} />;
```

Comment:
The fix could have been as simple as this one-line change. See https://webaim.org/techniques/skipnav/ it doesn't need a tabIndex. Reach UI doesn't have one either https://reach.tech/checkbox/.